util/util.go

```diff
@@ -787,3 +819,91 @@ func Contains(list []string, item string) bool {
 	}
 	return false
 }
+
+func GetReplicaDirectoryNames(diskPath string) (replicaDirectoryNames map[string]string, err error) {
+	defer func() {
+		err = errors.Wrapf(err, "cannot list replica directories in the disk %v", diskPath)
+	}()
+
+	replicaDirectoryNames = make(map[string]string, 0)
+
+	directory := filepath.Join(diskPath, "replicas")
+
+	initiatorNSPath := iscsi_util.GetHostNamespacePath(HostProcPath)
+	mountPath := fmt.Sprintf("--mount=%s/mnt", initiatorNSPath)
+	output, err := Execute([]string{}, "nsenter", mountPath, "ls", directory)
```

Comment:
Oh, btw, you add add `-maxdepth 1` and `-mindepth 1` to avoid unnecessary traversal.
```
find <Dir Path> -maxdepth 1 -mindepth 1 -type d -regextype posix-extended -regex ".*-[a-zA-Z0-9]{8}$"
```